PowerShellTools/Intellisense/IntelliSenseManager.cs

```diff
@@ -211,7 +215,12 @@ public int Exec(ref Guid pguidCmdGroup, uint nCmdId, uint nCmdexecopt, IntPtr pv
                     break;
 
                 case VSConstants.VSStd2KCmdID.COMPLETEWORD:
-
+                    isInStringArea = this.IsInStringArea(isInStringArea);
+                    if (isInStringArea == true)

```

Comment:
This is a bool? This way we'll have a compile error. Ref: https://msdn.microsoft.com/en-us/library/bb384091.aspx